main.go

```diff
@@ -130,8 +130,18 @@ func (p *gitHubPki) getTeamUsers() (err error) {
 	for _, team := range teams {
 		for _, t := range p.Config.Teams {
 			if *team.Name == t {
-				ghUsers, _, err := p.Client.Teams.ListTeamMembers(context.Background(), *team.ID, nil)
-				checkErr(err, "Failed to list team members for team "+*team.Name+": %v")
+				var ghUsers []*github.User
+				opt := &github.TeamListTeamMembersOptions{
+					ListOptions: github.ListOptions{PerPage: 100},
```

Comment:
If we're following pagination, do we really need to increase the amount of results per page?

Reply:
I used the same value as the listing of teams done above, but you are right, it's probably better to keep the default value. BTW, do you want me to update the listing of teams with the same changes as I did there to be consistent?

Reply:
Sure, good idea

Reply:
Done